test/functionalTest/cases/1217_update_compound_with_dots/update_compound_with_dots.test

```diff
@@ -0,0 +1,110 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Update compound attribute value with dots in keys
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Create entity
+# 02. Update entitiy attribute value with dots in keys
+# 03. Get entity
+#
+
+echo "01. Create entity"
+echo "================="
+payload='{
+  "type": "house",
+  "id": "apartment_3",
+  "temperature_2": {
+    "value": 40
+  }
+}'
+orionCurl --url /v2/entities --payload "$payload" --json
+echo
+echo
+
+
+echo "02. Update entitiy attribute value with dots in keys"
+echo "===================================================="
+payload='{
+  "temperature_2": {
+    "rt.ty": "5678"
+  }
+}'
+orionCurl --url '/v2/entities/apartment_3?options=keyValues' --payload "$payload" --json
```

Comment:
temperature_2 is a compound here, so the check of "dots in keys" might not be complete (compound values are 'special').
Why not add a new attribute called "rt.ty" ?